src/common-components/SocialAuthProviders.jsx

```diff
@@ -36,9 +37,11 @@ const SocialAuthProviders = (props) => {
       )
         : (
           <div className="font-container" aria-hidden="true">
-            <FontAwesomeIcon
-              icon={SUPPORTED_ICON_CLASSES.includes(provider.iconClass) ? ['fab', provider.iconClass] : faSignInAlt}
-            />
+            {SUPPORTED_ICON_CLASSES.includes(provider.iconClass) ? (
+              <FontAwesomeIcon icon={['fab', provider.iconClass]} />)
+              : (
+                <Icon className="h-75" src={Login} />
```

Comment:
Can we manage the icon position by reducing the padding from `font-container` instead using `h-75`?

Reply:
I have tried this but it also changes the position of `FontAwesomeIcon` which is placed inside the `font-container`. and also only the new Icon which is added needs to be fixed.